### DISCUSSION TITLE
2022-03-10-%E9%83%BD%E8%AF%B4%E5%86%99%E6%97%A5%E8%AE%B0%E6%98%AF%E5%A5%BD%E4%B9%A0%E6%83%AF%EF%BC%8C%E4%BD%86%E5%AD%A9%E5%AD%90%E4%B8%8D%E7%88%B1%E5%86%99%E6%80%8E%E4%B9%88%E5%8A%9E/

### DISCUSSION THREAD

**Discussion by giscus** [Fusyong/Fusyong.github.io#6] [Announcements]
# 2022-03-10-%E9%83%BD%E8%AF%B4%E5%86%99%E6%97%A5%E8%AE%B0%E6%98%AF%E5%A5%BD%E4%B9%A0%E6%83%AF%EF%BC%8C%E4%BD%86%E5%AD%A9%E5%AD%90%E4%B8%8D%E7%88%B1%E5%86%99%E6%80%8E%E4%B9%88%E5%8A%9E/

https://blog.xiiigame.com/2022-03-10-%E9%83%BD%E8%AF%B4%E5%86%99%E6%97%A5%E8%AE%B0%E6%98%AF%E5%A5%BD%E4%B9%A0%E6%83%AF%EF%BC%8C%E4%BD%86%E5%AD%A9%E5%AD%90%E4%B8%8D%E7%88%B1%E5%86%99%E6%80%8E%E4%B9%88%E5%8A%9E/

**Comment by fontomas** [comment]
孩子画得太有意思了！
以前尝试让我家孩子写日记，她总是写得很“糟糕”，或者写成周记、月记，但我也是最近才明白，日记不是做文章、不是绘画绣花，不需要那么雅致，日记是表达自己情绪的地方，是与自己对话的一种方式，只要自己高兴就好，何必拘泥于形式。

**Comment by Fusyong** [comment]
是，随意一点，回头看，也觉得有一种真趣。
孩子愿交流，愿思考，就很好。
